doc/developers/develop.rst

```diff
@@ -346,7 +346,8 @@ the correct interface more easily.
 And you can check that the above estimator passes all common checks::
 
     >>> from sklearn.utils.estimator_checks import check_estimator
-    >>> check_estimator(TemplateClassifier())  # passes
+    >>> check_estimator(TemplateClassifier())  # passes            # doctest: +SKIP
```

Comment:
I had to skip this one because the estimator is defined interactively and the  does not pickle ...
```
____________________________________________________________________________________________________________ [doctest] develop.rst _____________________________________________________________________________________________________________
341       ...         # Input validation
342       ...         X = validate_data(self, X, reset=False)
343       ...
344       ...         closest = np.argmin(euclidean_distances(X, self.X_), axis=1)
345       ...         return self.y_[closest]
346 
347 And you can check that the above estimator passes all common checks::
348 
349     >>> from sklearn.utils.estimator_checks import check_estimator
350     >>> check_estimator(TemplateClassifier())  # passes
UNEXPECTED EXCEPTION: PicklingError("Can't pickle <class 'TemplateClassifier'>: attribute lookup TemplateClassifier on builtins failed")
Traceback (most recent call last):
  File "/home/lesteve/micromamba/envs/scikit-learn-dev/lib/python3.13/doctest.py", line 1395, in __run
    exec(compile(example.source, filename, "single",
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                 compileflags, True), test.globs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<doctest develop.rst[10]>", line 1, in <module>
  File "/home/lesteve/dev/scikit-learn/sklearn/utils/_param_validation.py", line 216, in wrapper
    return func(*args, **kwargs)
  File "/home/lesteve/dev/scikit-learn/sklearn/utils/estimator_checks.py", line 857, in check_estimator
    check(estimator)
    ~~~~~^^^^^^^^^^^
  File "/home/lesteve/dev/scikit-learn/sklearn/utils/_testing.py", line 147, in wrapper
    return fn(*args, **kwargs)
  File "/home/lesteve/dev/scikit-learn/sklearn/utils/estimator_checks.py", line 2338, in check_estimators_pickle
    pickled_estimator = pickle.dumps(estimator)
_pickle.PicklingError: Can't pickle <class 'TemplateClassifier'>: attribute lookup TemplateClassifier on builtins failed
/home/lesteve/dev/scikit-learn/doc/developers/develop.rst:350: UnexpectedException
=====================================================================================================
```